cpp/src/utilities/device_atomics.cuh

```diff
@@ -450,20 +473,12 @@ template <typename T, typename BinaryOp>
 __forceinline__  __device__
 T genericAtomicOperation(T* address, T const & update_value, BinaryOp op)
 {
-    auto ret=  cudf::detail::genericAtomicOperationImpl<T, BinaryOp>{}
-            (address, update_value, op);
-    return T(ret);
-}
-
-// specialization for cudf::detail::wrapper types
-template <typename T, gdf_dtype dtype, typename BinaryOp, typename W = cudf::detail::wrapper<T, dtype> >
```

Comment:
This shouldn't be removed. I rely on it in https://github.com/rapidsai/cudf/pull/1478

It will need to be added back in a future PR.